uber/models.py

```diff
@@ -297,9 +297,11 @@ def _ints(self, name, val):
     @suffix_property
     def _label(self, name, val):
         try:
-            return self.get_field(name).type.choices[int(val)]
-        except KeyError:
-            return ''
+            val = int(val)
+        except ValueError:
```

Comment:
This is the new case we're handling, which is what happens when data gets munged. In fact, it's going to happen to Magstock once we switch over to the new campsite fields, because several attendees will have a string for site_number when we expect an int.